app/webpacker/src/stylesheets/_help.scss

```diff
@@ -0,0 +1,24 @@
+.govspeak .attachment,
+.gem-c-govspeak .attachment {
+  margin-top: 30px;
+  margin: 30px 0;
+  padding-top: 15px;
+  padding: 15px 0 0 129px;
+  position: relative;
+}
+
+.downloads {
+  section {
+    display: grid;
+    grid-template-columns: 20% 80%;
+    margin-bottom: 3em;
+  }
+
+  .attachment-thumb {
+    img {
+      border: 5px solid #f0f0f0;
+      max-width: 99px;
+      width: 100%;
```

Comment:
The text is wrapped over the thumbnail on moblie - you could drop the image size just on mobile using something like the following

```ruby
@include govuk-media-query($until: tablet) {
  max-width: 70px
}
```